ancestrymanager/ancestrymanager.go

```diff
@@ -331,6 +331,10 @@ func (m *manager) getProjectFromResource(d resources.TerraformResourceData, conf
 		res, ok = d.GetOk("project_id")
 		if ok {
 			return res.(string), nil
+		} 
+		res, ok = d.GetOk("parent")
```

Comment:
I think this is no longer needed because of the changes to the ancestry manager for Folder support. This will fall through to `getProjectFromSchema` which will parse out the parent.

Reply:
This makes sense. I saw the Info log about being unable to retrieve project_id and assumed it to be an error that I have to debug. I have reverted the change.